scripts/bluetooth/mesh/mesh_dfu_metadata.py

```diff
@@ -372,16 +373,32 @@ def encoded_metadata_get(version, comp):
     # FIX ME: Support user data
     return bytestring
 
+def input_parse():
+    parser = argparse.ArgumentParser(allow_abbrev=False)
+    parser.add_argument('--bin-path', required=True, type=str)
+    parser.add_argument('--print-metadata', action='store_true')
+    return parser.parse_known_args()[0]
+
+def existing_metadata_print(path):
+    try:
+        metadata_file = open(path, 'r')
+        print(f'Encoded metadata: {json.load(metadata_file).get("encoded_metadata")}')
```

Comment:
Now I feel that Ingar's suggestion about printing the entire JSON could make sense for this command at least. So that you get the complete metadata, not just the encoded part. This is just a thought, not a request (unless you and/or others feel the same).

Reply:
Well let me try it out. One thing comes to mind: What to do in the docs https://github.com/nrfconnect/sdk-nrf/pull/14078 with regard to command output (If we are doing this)? It will bloat the docs quite a bit to have an example of the entire composition.

Reply:
For example, use `..toggle::` https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/cheat_sheet.html#toggle
Or leave as is, or ask @mia-ko .